### ISSUE TITLE
Add riscv-64 to the supported architectures of the install script

### ISSUE THREAD

**Issue by afro-coder** [bytecodealliance/wasmtime#7580]
Thanks for filing a feature request! Please fill out the TODOs below.

#### Feature

TODO: Brief description of the feature/improvement you'd like to see in
Cranelift/Wasmtime.

The install script currently doesn't allow me to install a riscv-64 program

#### Benefit

TODO: What is the value of adding this in Cranelift/Wasmtime? What problems does
it solve?

Ease of installation for riscv devices
#### Implementation

TODO: Do you have an implementation plan, and/or ideas for data structures or
algorithms to use?

I'm unable to locate the install script curl `https://wasmtime.dev/install.sh -sSf | bash`

#### Alternatives

TODO: What are the alternative implementation approaches or alternative ways to
solve the problem that this feature would solve? How do these alternatives
compare to this proposal?

**Comment by afonso360** [comment]
I started working on this. Updating the script was fairly easy. However I've ran into an issue with the arch name being different in the releases vs what the OS identifies locally.

Locally I'm running Debian 13 (trixie), and `uname -m` identifies the architecture as `riscv64`, but on our release binaries we tag them as `riscv64gc`, which is the target triple name for our target.

I'm not entirely sure where we should fix this. Do we update our release names, or should I try to fix this up in the script? (cc: @alexcrichton)
